homeassistant/components/bryant_evolution/config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""Config flow for Bryant Evolution integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from evolutionhttp import BryantEvolutionClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlowResult
+from homeassistant.const import CONF_HOST
+
+from .const import CONF_SYSTEM_ID, CONF_ZONE_ID, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_SYSTEM_ID, default=1): vol.All(int, vol.Range(min=1)),
+        vol.Required(CONF_ZONE_ID, default=1): vol.All(int, vol.Range(min=1)),
```

Comment:
What are those values?

Reply:
which values? these are the system ID and zone ID of the HVAC equipment -- did you want a comment?

Reply:
I think the question is more: are those easily recognizable/identifyable by the user, as in: Do they know what they need to put there?

Reply:
Yes, I believe they do.